mpf/platforms/fast/communicators/net_neuron.py

```diff
@@ -294,6 +294,9 @@ def update_switches_from_hw_data(self):
         it will process it like any switch change.
         """
         for switch in self.machine.switches.values():
+            if switch.platform != self.platform:
```

Comment:
I expect there's a better way in python to filter, and then not even need the continue

Reply:
Took a few whacks at it and found the final syntax. It's a little awkward imo, but it's in :)